pkgs/applications/audio/raysession/default.nix

```
@@ -35,7 +36,10 @@ buildPythonApplication rec {
   ];
 
   postFixup = ''
-    wrapPythonProgramsIn "$out/share/raysession/src" "$out $pythonPath"
+    wrapPythonProgramsIn "$out/share/raysession/src" "$out $pythonPath" "$out/bin"
```

```suggestion
    wrapPythonProgramsIn "$out/share/raysession/src" "$out $pythonPath"
```
It seems that this parameter is unnecessary. 
https://github.com/NixOS/nixpkgs/blob/58e6faecd51ea396019419764eaf43ba993a8ba7/pkgs/development/interpreters/python/wrap.sh#L42-L45